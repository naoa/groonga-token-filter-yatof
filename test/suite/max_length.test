register token_filters/yatof

tokenize TokenBigram \
  "abcde abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz" \
  --normalizer NormalizerAuto \
  --token_filters TokenFilterMaxLength
