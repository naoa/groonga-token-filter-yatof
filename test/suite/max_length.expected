register token_filters/yatof
[[0,0.0,0.0],true]
tokenize TokenBigram   "abcde abcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyzabcdefghijklmnopqrstuvwxyz"   --normalizer NormalizerAuto   --token_filters TokenFilterMaxLength
[[0,0.0,0.0],[{"value":"abcde","position":0}]]
